data/reusables/files/choose-commit-email.md

```diff
@@ -1,4 +1,4 @@
 {% if currentVersion == "free-pro-team@latest" %}
-1. Below the commit message fields, click the email address drop-down menu and choose a Git author email address. Only verified email addresses appear in this drop-down menu. If you enabled email address privacy, then `<username>@users.noreply.github.com` is the default commit author email address.  For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
+1. If creating a merge commit, click the email address drop-down menu below the commit message fields and choose a Git author email address. Only verified email addresses appear in this drop-down menu. If you enabled email address privacy, then `<username>@users.noreply.github.com` is the default commit author email address.  For more information, see "[Setting your commit email address](/articles/setting-your-commit-email-address)."
```

Comment:
As a user, I still don't think this would enough to explain why squash-merge commits don't see the email selector. Squash merges still result in a :airquotes: merge commit :airquotes:, so would it be easier to just say that the email selector isn't available for squash merges because <explanation of how we credit squash merges>?